upcloud/client/client.go

```diff
@@ -148,6 +148,18 @@ func WithBaseURL(baseURL string) configFn {
 	}
 }
 
+func WithInsecureSkipVerify() configFn {
+	return func(c *config) {
+		if c.httpClient != nil {
+			c.httpClient.Transport = &http.Transport{
+				TLSClientConfig: &tls.Config{
+					InsecureSkipVerify: true, //nolint
```

Comment:
While easy to understand given the function, this could have a comment explaining why `//nolint` is used. Also wondering if `//nolint` now also disables `nolintlint` linter here, which should give error about `//nolint` without comment  😅

Reply:
Replaced with `nosec` and addded a comment! Makes more sense now.